R/zchunk_L2322.Fert.R

```diff
@@ -76,272 +76,447 @@ module_energy_L2322.Fert_DISABLED <- function(command, ...) {
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("common/GCAM_region_names", "energy/A322.sector") ->
       L2322.Supplysector_Fert
 
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L2322.FinalEnergyKeyword_Fert %>%
+      add_title("Supply sector keywords for fertilizer sector") %>%
+      add_units("NA") %>%
+      add_comments("Expanded fertilizer sector supply sector keywords for all GCAM regions") %>%
       add_legacy_name("L2322.FinalEnergyKeyword_Fert") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("common/GCAM_region_names", "energy/A322.sector") ->
       L2322.FinalEnergyKeyword_Fert
 
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L2322.SubsectorLogit_Fert %>%
+      add_title("Subsector logit exponents of fertilizer sector") %>%
+      add_units("NA") %>%
+      add_comments("Expanded fertilizer sector subsector logit exponents for all GCAM regions") %>%
       add_legacy_name("L2322.SubsectorLogit_Fert") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("energy/A322.subsector_logit", "common/GCAM_region_names") ->
       L2322.SubsectorLogit_Fert
 
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L2322.SubsectorShrwt_Fert %>%
+      add_title("Subsector shareweights of fertilizer") %>%
+      add_units("NA") %>%
+      add_comments("Created as empty table") %>%
       add_legacy_name("L2322.SubsectorShrwt_Fert") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("energy/A322.subsector_shrwt", 'common/GCAM_region_names') ->
       L2322.SubsectorShrwt_Fert
 
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L2322.SubsectorShrwtFllt_Fert %>%
+      add_title("Subsector shareweights of fertilizer") %>%
+      add_units("Unitless") %>%
+      add_comments("Expanded Subsector shareweights of fertilizer to all GCAM regions") %>%
```

Comment:
See above. These ones are more specific about what is in them, but still need information on where it came from.